apps/mempool/docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+services:
+  mempool:
+    container_name: mempool
+    image: mempool/frontend:v3.0.0-dev2
+    restart: unless-stopped
+    stop_grace_period: 1m
+    user: 0:0
+    command: "nginx -g 'daemon off;'"
+    ports:
+      - 2200:8080
+    depends_on:
+      - mempool-db
+    environment:
+      FRONTEND_HTTP_PORT: "8080"
+      BACKEND_MAINNET_HTTP_HOST: "mempool-api"
+    networks:
+      - tipi_main_network
+    labels:
+      # Main
+      traefik.enable: true
+      traefik.http.middlewares.mempool-web-redirect.redirectscheme.scheme: https
+      traefik.http.services.mempool.loadbalancer.server.port: 8080
+      # Web
+      traefik.http.routers.mempool-insecure.rule: Host(`${APP_DOMAIN}`)
+      traefik.http.routers.mempool-insecure.entrypoints: web
+      traefik.http.routers.mempool-insecure.service: mempool
+      traefik.http.routers.mempool-insecure.middlewares: mempool-web-redirect
+      # Websecure
+      traefik.http.routers.mempool.rule: Host(`${APP_DOMAIN}`)
+      traefik.http.routers.mempool.entrypoints: websecure
+      traefik.http.routers.mempool.service: mempool
+      traefik.http.routers.mempool.tls.certresolver: myresolver
+      # Local domain
+      traefik.http.routers.mempool-local-insecure.rule: Host(`mempool.${LOCAL_DOMAIN}`)
+      traefik.http.routers.mempool-local-insecure.entrypoints: web
+      traefik.http.routers.mempool-local-insecure.service: mempool
+      traefik.http.routers.mempool-local-insecure.middlewares: mempool-web-redirect
+      # Local domain secure
+      traefik.http.routers.mempool-local.rule: Host(`mempool.${LOCAL_DOMAIN}`)
+      traefik.http.routers.mempool-local.entrypoints: websecure
+      traefik.http.routers.mempool-local.service: mempool
+      traefik.http.routers.mempool-local.tls: true
+
+
+  mempool-api:
+    container_name: mempool-api
+    image: mempool/backend:v3.0.0-dev2
+    restart: unless-stopped
+    stop_grace_period: 1m
+    user: 0:0
+    volumes:
+      - ${BITCOIND_DIR:-${APP_DATA_DIR}/../bitcoind/data}:/data/.bitcoin:ro
```

Comment:
is it working as expected ?

Reply:
I've tested this out, it works as expected. Either the value of `BITCOIND_DIR` is used if set, or by default your `app-data/bitcoind/data` folder is used.